src/io/flutter/dart/DartSdkLibUtilDelegate.java

```diff
@@ -94,7 +94,7 @@ public void ensureDartSdkConfigured(@Nullable Project project, String sdkHomePat
       }
       else {
         //noinspection unchecked
-        final Method method = delegate.getMethod("ensureDartSdkConfigured", Project.class, Collection.class);
+        final Method method = delegate.getMethod("ensureDartSdkConfigured", Project.class, String.class);
```

Comment:
This was an error on my part in the method params. Pre 2017.1, this method in the dart plugin took one param; in 2017.1+ it takes two.